.github/workflows/conformance.yaml

```diff
@@ -0,0 +1,37 @@
+on:
+  push:
+    branches:
+    - main
+    paths:
+      - 'google-cloud-bigtable/**'
+  pull_request:
+    paths:
+      - 'google-cloud-bigtable/**'
+name: conformance
+jobs:
+  conformance:
```

Comment:
Can you use a job name that includes the word "bigtable" (e.g. `bigtable_conformance` or similar) so that it's clear in the github UI which client's tests are running?